docs/concepts-new/mesh.md

```diff
@@ -4,31 +4,55 @@ title: Introduction to Temporal's core concepts
 sidebar_label: Introduction
 ---
 
-:::note
+:::caution
 
 This page is a work in progress!
 
 :::
 
-**Welcome to Temporal's core terminology!**
+**Temporal's tenth rule**
+
+> Any sufficiently complex distributed system contains an ad-hoc, informally-specified, bug-ridden, slow implementation of half of temporal.io.
+
+## What is Temporal?
+
+[temporal.io](https://temporal.io) is a scalable and reliable runtime for Temporal Workflow Executions.
```

Comment:
Including the website URL here seems misleading because the runtime doesn't live there. Change to simply "Temporal"?